frontend/packages/ux-editor/src/components/config/expressions/ExpressionContent.tsx

```diff
@@ -0,0 +1,159 @@
+  }
+
+  const addTriggerDataSource = (dataSource: string) => {
+    expressionElement.dataSource = dataSource;
+    handleUpdateExpressionElement();
+  };
+
+  const specifyTriggerDataSource = (dataSourceKind: string) => {
+    expressionElement.value = dataSourceKind;
+    handleUpdateExpressionElement();
+  };
+
+  const addComparableTriggerDataSource = (compDataSource: string) => {
+    expressionElement.comparableDataSource = compDataSource;
+    handleUpdateExpressionElement();
+  };
+
+  const specifyComparableTriggerDataSource = (compDataSourceKind: string) => {
+    expressionElement.comparableValue = compDataSourceKind;
+    handleUpdateExpressionElement();
+  };
+
+  const addExpressionOperatorForPrevExpression = (operator: 'Og' | 'Eller') => {
+    expressionElement.expressionOperatorForPrevExpression = operator;
+    handleUpdateExpressionElement();
+  };
+
+  const handleAddExpressionElement = () => {
+    setShowAddExpressionButton(!showAddExpressionButton);
+    expressionElement.expressionOperatorForPrevExpression = 'Og';
+    onAddExpressionElement();
+  }
+
+  const handleUpdateExpressionElement = () => {
+    onUpdateExpressionElement(expressionElement);
+  };
+
+  const handleRemoveExpressionElement = () => {
+    onRemoveExpressionElement(expressionElement);
+  }
+
+  return (
+    <div>
+      <span>Velg oppsettet for uttrykket du vil lage</span>
+      <Select
+        onChange={(func: string) => addFunctionToExpressionElement(func)}
+        options={Object.values(ExpressionFunction).map((func: string) => ({ label: func, value: func }))}
+        value={expressionElement.function || 'Velg oppsett...'}
+        searchLabel={'Velg...'}
+      />
+      {allowToSpecifyExpression &&
+        <div>
+          <Button
+            color={ButtonColor.Danger}
+            icon={<XMarkIcon />}
+            onClick={handleRemoveExpressionElement}
+            variant={ButtonVariant.Quiet}
+          />
+          <Select
+            onChange={(dataSource: string) => addTriggerDataSource(dataSource)}
+            options={[
+              { label: 'komponent', value: 'Komponent' },
+              { label: 'datamodell', value: 'Datamodell' }
+            ]}
+            value={expressionElement.dataSource || 'Velg...'} // Is it necessary with the first check?
+            searchLabel={'Velg...'}
+          />
+          {expressionElement.dataSource &&
+            <Select
+              onChange={(dataSourceKind: string) => specifyTriggerDataSource(dataSourceKind)}
+              options={[
+                { label: 'alder', value: 'Alder' },
+                { label: 'fornavn', value: 'Fornavn' }
+              ]}
+              value={expressionElement.value || 'Velg...'}
+              searchLabel={'Velg...'}
+            />}
+          <span>{expressionElement.function}</span>
+          <Select
+            onChange={(compDataSource: string) => addComparableTriggerDataSource(compDataSource)}
```

Comment:
I guess this is something that might come later on in the implementation, but the compare source/value should probably also include just typing a custom value.